include/grpc++/impl/codegen/thrift_serializer.h

```diff
@@ -131,12 +131,12 @@ class ThriftSerializer {
     last_deserialized_ = true;
 
     // reset buffer transport
-    buffer_->resetBuffer((uint8_t*)serialized_buffer, length);
+    buffer_->resetBuffer(const_cast<uint8_t*>(serialized_buffer), length);
```

Comment:
That still doesn't answer the question, though, of why should Deserialize take a `const uint8_t*` as its argument if it is just going to strip the const-ness?